src/osg/TextureRectangle.cpp

```diff
@@ -202,10 +202,10 @@ void TextureRectangle::apply(State& state) const
         {
             _subloadCallback->subload(*this, state);
         }
-        else if (_image.valid() && getModifiedCount(contextID) != _image->getModifiedCount())
+        else if (_image.valid() && _modifiedCount[contextID] != _image->getModifiedCount())
         {
             // update the modified count to show that it is up to date.
-            getModifiedCount(contextID) = _image->getModifiedCount();
+            _modifiedCount[contextID] = _image->getModifiedCount();
```

Comment:
this is one example where getModified() had to return a ref (and does not have to anymore)